website/docs/r/bgp_route_summarization.html.markdown

```diff
@@ -11,32 +11,38 @@ description: |-
 
 Manages ACI BGP Route Summarization
 
-## Example Usage
+## API Information ##
 
-```hcl
-resource "aci_bgp_route_summarization" "example" {
+* `Class` - bgpRtSummPol
+* `Distinguished Name` - uni/tn-{tenant_name}/bgprtsum-{bgp_rt_summ_pol_name}
+
+## GUI Information ##
 
-  tenant_dn   = aci_tenant.example.id
-  name        = "example"
-  annotation  = "example"
-  description = "from terraform"
-  attrmap     = "example"
-  ctrl        = "as-set"
-  name_alias  = "example"
+* `Location` - Tenants -> Policies -> Protocol -> BGP -> BGP Route Summarization
 
+## Example Usage
+
+```hcl
+resource "aci_bgp_route_summarization" "bgp_rt_summ_pol" {
+  tenant_dn             = aci_tenant.tf_tenant.id
+  name                  = "bgp_rt_summ_pol"
+  description           = "from terraform"
+  attrmap               = "sample attrmap"
+  ctrl                  = sort(["summary-only", "as-set"])                 # Use the sorted list to handle identical changes
```

Comment:
Remove comments # Use the sorted list to handle identical changes and # Use the sorted list to handle identical changes